Extras/FrontendTests/tests/keyboard.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { test } from './fixtures';
+import { expect } from './matchers';
+import {
+    PSEventTypes,
+    DataChannelEvent,
+    getEventsFor,
+} from './extras';
+import * as helpers from './helpers';
+
+test('Test keyboard events', {
+    tag: ['@keyboard'],
+}, async ({ page, streamerPage, streamerId }) => {
+
+    await page.goto(`/?StreamerId=${streamerId}&MatchViewportRes=true`);
+    await page.getByText('Click to start').click();
+    await helpers.waitForVideo(page);
+
+    const player_box = await page.locator('#videoElementParent').boundingBox();
```

Comment:
Any particular reason for `underscored_variables` - seems a departure from the rest of the codebase?

Reply:
It was picked up from somewhere. Maybe when I started doing playwright tests, and was trying to be consistent and it became a habit when moving into this code base.
The joys of trying to be consistent with everyone.